x-pack/plugins/observability_solution/inventory/public/plugin.tsx

```diff
@@ -7,6 +7,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { i18n } from '@kbn/i18n';
+import { from } from 'rxjs';
+import { map } from 'rxjs';
```

Comment:
nit: these imports can be merged into one.

Reply:
addressed [here](https://github.com/elastic/kibana/pull/193251/commits/ff54d98ca4dd7f8a12040b622bb8cb82ffe1dcd7)